js/views/pauseScreen.js

```diff
@@ -104,41 +108,20 @@ class PauseScreen extends React.Component {
     this.props.handleVrPlayerClick();
   }
 
-  /**
-   * call handleTouchEnd when touchend was called on selectedScreen and videoType is Vr
-   * @param {Event} e - event object
-   */
-  handleTouchEnd(e) {
-    if (this.props.controller.state.controlBarVisible) {
-      var shouldToggle = false;
-      if (this.props.controller.videoVr) {
-        e.preventDefault();
-        if (!this.props.isVrMouseMove) {
-          shouldToggle = true;
-        }
-      } else {
-        shouldToggle = true;
-      }
-      if (shouldToggle) {
-        this.props.controller.togglePlayPause(e);
-      }
-    }
-  }
-
   /**
    * call handleVrTouchEnd when touchend was called on selectedScreen and videoType is Vr
```

Comment:
Guess we don't need this method. Just bypass this.props.handleVrPlayerMouseUp to the 52 and 61 line where you use it. If required binding, arrow functions are much shorter way to get it.

Reply:
\+

Reply:
\+